spec/classes/init_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'ssh' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:params) { {} }
```

Comment:
AFAIK this is unneeded.

Reply:
Just confirmed, and you are correct. Removed.